app/models/models.py

```diff
@@ -17,7 +21,7 @@ class Village(db.Base):
     __tablename__ = "villages"
     id = Column(String(255), primary_key=True, index=True, default=uuid4().hex)
     name = Column(String(255), index=True)
-    loation = Column(String(255), ForeignKey("locations.id"))
+    location_id = Column(String(255), ForeignKey("locations.id"))
```

Comment:
You are certain that the tablename is called `locations`, to enable code readability i would suggest using the format `ForeignKey(Location.id)` as opposed to `ForeignKey("locations.id")`